src/components/ui/Alert/alert.scss

```diff
@@ -0,0 +1,32 @@
+[data-store-alert] {
+  box-sizing: border-box; /** FIXME remove after add css reset */
+  display: flex;
+  align-items: center;
+  min-height: var(--space-7);
+  padding: var(--space-1) var(--space-3);
+  background-color: var(--bg-primary-light);
+  color: var(--color-text);
+  font-size: var(--text-size-1);
+
+  [data-store-icon] {
+    margin: 0;
+    padding-right: var(--space-1);
+  }
+
+  span { font-weight: bold; }
+
+  a {
+    font-weight: bold;
+    color: var(--color-text);
+
+    &:visited { color: var(--color-text) }
+  }
+
+  [data-alert-button] {
```

Comment:
Btw, I'm not sure if we are going to keep the `button[data-store-button]`  width/height: `48px`. If not, we might need to add width and height to this button too.

Reply:
That's true. 
When I added the `min-width/min-height: 48px (var(--space-7))` the CSS was broken. But we can fix using.

```css
[data-store-alert] {
  position: relative;
  
  [data-alert-button] {
    position: absolute;
    right: 0;
  }
}
```